jest.config.js

```diff
@@ -1,20 +1,11 @@
 module.exports = {
   testEnvironment: 'node',
   transform: {
-    "^.+\\.tsx?$": "ts-jest"
+    '^.+\\.tsx?$': 'ts-jest',
   },
-  moduleFileExtensions: [
-    "ts",
-    "tsx",
-    "js",
-    "jsx",
-    "json",
-    "node",
-  ],
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
   testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.(ts|js)x?$',
   coverageDirectory: 'coverage',
-  collectCoverageFrom: [
-    'src/**/*.{ts,tsx,js,jsx}',
-    '!src/**/*.d.ts',
-  ],
+  coveragePathIgnorePatterns: ['src/openapi/'],
+  collectCoverageFrom: ['src/**/*.{ts,tsx,js,jsx}', '!src/**/*.d.ts'],
```

Comment:
Not a problem, but specifying we want coverage from tsx and jsx files is unnecessary since we're unlikely to be writing React or Preact components in the node SDK.
```suggestion
  collectCoverageFrom: ['src/**/*.{ts,js}', '!src/**/*.d.ts'],
```

Reply:
👍